src/app/component/auth/sign-in-new/sign-in-new.component.html

```diff
@@ -5,25 +5,61 @@
          alt="sign-in image">
   </div>
   <div class="right-side">
-    <a (click)="closeSignInWindow()"
+    <a (click)="CloseSignInWindow()"
```

Comment:
add "on" for all click events. It more readable-> "onCloseSignInWindow()"

Reply:
Actually it`s not a requirement to add "on"